README.md

```diff
@@ -53,12 +53,17 @@ docker run -p 8081:80 fossology/fossology
 
 The docker image can then be used using http://IP_OF_DOCKER_HOST:8081/repo user fossy password fossy.
 
-Execution with external database container can be done using Docker Compose, via the following command:
+
+If you want to run Fossology with an external database container, you can use Docker Compose, via the following command: 
 
 ```sh
 docker-compose up
 ```
 
+Docker Compose is a tool that allows you to define and run multi-container applications using a YAML file. FOSSology provides a docker-compose.yml file that defines three services: scheduler, web, and db. The scheduler service runs the FOSSology scheduler daemon, which handles the analysis tasks. The web service runs the FOSSology web server, which provides the web interface. The db service runs a PostgreSQL database server, which stores the FOSSology data. You can use the docker-compose up command to start all three services at once.
+
+The Fossology web service allows you to configure its database connection using some environment variables. These variables are defined in the docker-compose.yml file under the environment key.
```

Comment:
Please avoid writing long lines. It becomes very difficult to review them. Please add line breaks at every 80 characters or so.